sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -131,6 +145,12 @@
      */
     private AnalyticsListener mAnalyticsListener;
 
+    /**
+     * Transmission interval.
+     * Valid value ranges from 3 seconds to 1 day.
+     */
+    private int mTransmissionInterval;
```

Comment:
The value itself isn't actually transmission interval but trigger interval. Should we just say `mTriggerInterval` for this private variable? Maybe we can also say `setTriggerInterval` for the instance method and only say "transmission interval" for public facing API but it isn't consistent as well. @guperrot What's your opinion?

Reply:
Trigger interval is not in any specification, I think this is fine to use the new name here.